coveralls/api.py

```diff
@@ -76,67 +83,71 @@ def ensure_token(self):
     @staticmethod
     def load_config_from_appveyor():
         pr = os.environ.get('APPVEYOR_PULL_REQUEST_NUMBER')
-        return 'appveyor', os.environ.get('APPVEYOR_BUILD_ID'), pr
+        return 'appveyor', os.environ.get('APPVEYOR_BUILD_ID'), None, pr
 
     @staticmethod
     def load_config_from_buildkite():
         pr = os.environ.get('BUILDKITE_PULL_REQUEST')
         if pr == 'false':
             pr = None
-        return 'buildkite', os.environ.get('BUILDKITE_JOB_ID'), pr
+        return 'buildkite', os.environ.get('BUILDKITE_JOB_ID'), None, pr
 
     @staticmethod
     def load_config_from_circle():
         pr = os.environ.get('CI_PULL_REQUEST', '').split('/')[-1] or None
-        return 'circle-ci', os.environ.get('CIRCLE_BUILD_NUM'), pr
+        return 'circle-ci', os.environ.get('CIRCLE_BUILD_NUM'), None, pr
 
     @staticmethod
     def load_config_from_github():
-        service_number = os.environ.get('GITHUB_SHA')
+        service = 'github'
+        if 'COVERALLS_REPO_TOKEN' in os.environ:
```

Comment:
Could you modify this line to be `if os.environ.get('COVERALLS_REPO_TOKEN'):`? I've run into some systems before where `foo in os.environ` always returns True for unset environment variables (eg. they get set to `os.environ[foo] = None`).